torchvision/transforms/functional_tensor.py

```diff
@@ -226,8 +226,8 @@ def adjust_gamma(img: Tensor, gamma: float, gain: float = 1) -> Tensor:
 
     result = (gain * result ** gamma).clamp(0, 1)
 
-    result = convert_image_dtype(result, dtype)
-    result = result.to(dtype)
+    if result.dtype != dtype:
```

Comment:
```suggestion
```
As [discussed earlier](https://github.com/pytorch/vision/pull/3471#discussion_r583989827), this check is not needed because it's already included in the first line of `convert_image_dtype()`.